pkg/virt-handler/device-manager/deviceplugin/v1beta1/api.proto

```diff
@@ -124,7 +161,16 @@ message AllocateRequest {
 }
 
 message ContainerAllocateRequest {
-	repeated string devicesIDs = 1;
+	repeated string devices_ids = 1 [(gogoproto.customname) = "DevicesIDs"];
+}
+
+// CDIDevice specifies a CDI device information.
+message CDIDevice {
```

Comment:
Nit: in KubeVirt we also have a project called [CDI](https://github.com/kubevirt/containerized-data-importer), please make a clear distinction that this is not the Containerized Data Importer